server/aesdsocket-start-stop.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
```

Comment:
@neohere97 you shouldn't check your aesdsocket file into your repo.  Add this to gitignore.

Reply:
I had actually caught it, forgot to put it into gitignore